libs/banglejs/jswrap_bangle.c

```diff
@@ -1562,6 +1574,7 @@ void backlightOffHandler() {
 #endif // !EMULATED
 
 void btnHandlerCommon(int button, bool state, IOEventFlags flags) {
+  bool pushEvent = true;
```

Comment:
Is this var used?

Reply:
Yes, I thought it made more sense to not use `return` statements in this function so that the `if (button == lcdWakeButton) {` code can be reached.  `if (bangleFlags&JSBF_LOCKED) pushEvent = false;` instead of `if (bangleFlags&JSBF_LOCKED) return` .  Not super important but makes the feeling of writing/changing the function more stress-free?